### DISCUSSION TITLE
not works for me

### DISCUSSION THREAD

**Discussion by Roghayeh-Dehghani** [SaeedDev94/ng-persian-datepicker#60] [Q&A] [answered]
Hi
I have used this date Picker but don't show in runtime without any error!
 
<ng-persian-datepicker>
                        <input type="text" [formControl]="dateValue" style="width: 6rem;" />
                    </ng-persian-datepicker>

if replace with the bellow part it works fine:
<input type="text" #datepickerInput />
<ng-persian-datepicker [input]="datepickerInput">

**Comment by SaeedDev94** [comment, marked as answer]
@Roghayeh-Dehghani 
Hi
Which version are you using?
Latest version, currently `v5.0.0`, changed a lot !
This is the old approach of using `ng-persian-datepicker`,
Please read `README.md` file again,
Thanks!

**Reply by Roghayeh-Dehghani** [reply]
i installed it today

**Reply by SaeedDev94** [reply]
So you should probably have `v5.0.0` (check your `package.json` file to find out the lib version)
And the way you are trying to use it is completely wrong (for `v5.0.0`)
You can either read [README.md](https://github.com/Saeed-Pooyanfar/ng-persian-datepicker/blob/master/README.md) file or check [demo](https://github.com/Saeed-Pooyanfar/ng-persian-datepicker/tree/master/src/app/demo) source code

**Reply by Roghayeh-Dehghani** [reply]
thank you, it works fine :)
but i want to show date in format "YYYY/MM/DD" and also datepicker  push items in the form to bellow.

**Reply by SaeedDev94** [reply]
you should disable time, and set dateFormat to 'jYYYY-jMM-jDD'  
for pushing items you should create relative => absolute containers to make it float (via global styles)

**Reply by Roghayeh-Dehghani** [reply]
thanks a lot, yes i did it :)
